lib/services/swagger-api-service.js

```diff
@@ -207,7 +210,12 @@ function swaggerFactory(
             res.status(status).send(data);
         })
         .catch(function(err) {
-            next(_processError(err));
+            if (res.locals.errorStatus) {
+                next();
```

Comment:
@dalebremner What are we handling off to from the error handler fitting?  I think the error handler should always close the request in some way, so I would think that calling next will complain that headers have already been sent.  Should we be doing something like calling `res.status(500).json(someDefaultError)`